sources/Makefile

```diff
@@ -76,12 +76,16 @@ fm-simplex:
 	$(DUNE) build $(DUNE_FLAGS) \
 		_build/install/default/share/alt-ergo/plugins/fm-simplex-plugin.cma \
 		_build/install/default/share/alt-ergo/plugins/fm-simplex-plugin.cmxs
+		ln -sf _build/install/default/share/alt-ergo/plugins/fm-simplex-plugin.cma fm-simplex-plugin.cma
+		ln -sf _build/install/default/share/alt-ergo/plugins/fm-simplex-plugin.cmxs fm-simplex-plugin.cmxs
```

Comment:
Wrong indentation, it should start at the same column as the `$(DUNE)` a few lines above